srcpkgs/nodejs-lts-10/template

```diff
@@ -37,9 +37,7 @@ if [ "$XBPS_WORDSIZE" -ne "$XBPS_TARGET_WORDSIZE" ]; then
 	nocross="host and target must have the same pointer size"
 fi
 
-if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
-	makedepends+=" libatomic-devel"
-fi
+makedepends+=" libatomic-devel"
```

Comment:
Missing the comment I suggested.

Reply:
You should probably just add it to the initial `makedepends` list.

Reply:
Oh, you meant in the template itself.

Reply:
Yeah `node` and `node-lts` just have it in `makedepends` for all architectures. I'll do that.